eng/Versions.props

```diff
@@ -38,7 +38,7 @@
   </PropertyGroup>
   <PropertyGroup>
     <!-- Dependency from https://github.com/dotnet/templating -->
-    <MicrosoftDotNetCommonItemTemplatesPackageVersion>7.0.100-rc.1.22404.7</MicrosoftDotNetCommonItemTemplatesPackageVersion>
+    <MicrosoftDotNetCommonItemTemplatesPackageVersion>7.0.100-dev</MicrosoftDotNetCommonItemTemplatesPackageVersion>
```

Comment:
This can't be good. @marcpopMSFT is this caused by parts of the templating repo moving into the SDK repo?

Reply:
cc @GangWang01

Reply:
Thanks @eerhardt we are currently working on the fix

Reply:
We fixed couple of things here: https://github.com/dotnet/installer/pull/14295, hopefully next insertion is correct.